### PR TITLE
Fix the user document

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/docs/contribution-guidelines/control-plane-development.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ description: >
 Prepare a ControlPlane configuration file as described at [Installation](https://pipecd.dev/docs/operator-manual/control-plane/installation/) and start running by the following command:
 
 ``` console
-bazelisk run //cmd/server:server -- server \
+bazelisk run //cmd/pipecd -- server \
```
Comment (cakecatz) [inline docs/content/en/docs/contribution-guidelines/control-plane-development.md:23]: I omitted it, but should I write it properly like `pipecd:pipecd`?

Comment (khanhtc1202) [reply docs/content/en/docs/contribution-guidelines/control-plane-development.md:23]: yes, I think we should 🙏

Comment (cakecatz) [reply docs/content/en/docs/contribution-guidelines/control-plane-development.md]: I updated 😄

Comment (khanhtc1202) [reply docs/content/en/docs/contribution-guidelines/control-plane-development.md]: thanks 🙏

**Comment by khanhtc1202** [comment]
🏃 
/approve

**Pull request merged**